### ISSUE TITLE
Atom sinks

### ISSUE THREAD

**Issue by ebuchman** [cosmosdevs/atom2021#9]
I think part of the story for Atom value accrual should be consideration of Atom sinks. We have a significant source in the inflation but the only sink we have is slashing which is very minimal and infrequent. It might be valuable to consider a more regular sink for the token. Obviously this parallels the EIP1559 discussion happening in ETH.

We could imagine implementing something similar for Atoms. We could use an onchain AMM so that folks can still offer the fee in other coins, and have it pass to the AMM to get Atoms which are burned for the tx to be valid. cc https://github.com/cosmosdevs/atom2021/issues/5#issuecomment-672852656

There could be other opportunities for sinks, including setting up shared security and other IBC constructions.

**Comment by Hyung-bharvest** [comment]
I don't think such microeconomic demand/supply governs the price movement. If we have good number of incoming user-bases from different blockchains to enjoy our utility, price will jump in multiples ignoring any micro demand/supply environment. I think it would be better to focus on attracting users from external blockchain space than thinking about tweaking internal microeconomic structure.

The consequence of price not appreciating is coming from "no enduser utility" from the Hub for 1.5 years of mainnet. I don't think it is caused by inflation or minor necessary inflow of capital.

**Comment by ebuchman** [comment]
I agree in the short term this doesn't matter much, and what matters is actual utility from interop. But in the mid/longer term this may be important to have a sound foundation for fees with better UX. Also the exponential inflation of Atom may not be sustainable or competitive as other hubs come online, so the monetary policy may start to matter more.
